tests/foreman/cli/test_oscap.py

```diff
@@ -475,6 +475,10 @@ def test_postive_create_scap_policy_with_valid_name(self, name, scap_content):
             }
         )
         assert scap_policy['name'] == name
+        # Deleting policy which created for all valid input (ex- latin1, cjk, utf-8, etc.)
+        Scappolicy.delete({'name': scap_policy['name']})
+        with pytest.raises(CLIReturnCodeError):
+            Scapcontent.info({'name': scap_policy['name']})
```

Comment:
```suggestion
        target_sat.cli.Scappolicy.delete({'name': scap_policy['name']})
        with pytest.raises(CLIReturnCodeError):
            target_sat.cli.Scapcontent.info({'name': scap_policy['name']})
```

Reply:
successfully test the changes suggested above, but I will request not to incorporate those changes now, as we are going to refactor oscap component in upcoming days.